bot/exts/info/pep.py

```diff
@@ -97,9 +97,12 @@ async def validate_pep_number(self, pep_nr: int) -> Optional[Embed]:
 
     def generate_pep_embed(self, pep_header: Dict, pep_nr: int) -> Embed:
         """Generate PEP embed based on PEP headers data."""
+        # the parsed header can be wrapped to multiple lines, so we need to make sure that is removed
+        # for an example of a pep with this issue, see pep 500
+        title = " ".join([line.strip() for line in pep_header["Title"].split()])
```

Comment:
is the strip necessary? split should collapse whitespace

Reply:
https://github.com/python-discord/bot/pull/1994/commits/af534ce297f68aedf6aa5a59f82a539c6cbd8686